### PR TITLE
Scf 4 first meeting minutes blog

### PR DESCRIPTION
**Pull request by karinlag**
Adding blog post regarding minutes from first four meetings.

### CODE REVIEW

**Comment by gvwilson** [comment]
Is this one ready to merge?

File: blog/2015/04/SCF_minutes.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+root: ../../..
+author: Karin Lagesebn
+title: "The Steering Committee has landed!"
+date: 2015-04-02
+time: "10:00:00"
+category: ["Software Carpentry Foundation"]
+---
+<!-- start excerpt -->
+<p>
+The Steering Committe has now had its first few meeetings. Minutes from them are up, please go have a look if you are wondering what we are up to!
```
Comment (r-gaia-cs) [inline blog/2015/04/SCF_minutes.html]: Can we add a link?

File: blog/2015/04/SCF_minutes.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+layout: blog
+root: ../../..
+author: Karin Lagesebn
```
Comment (r-gaia-cs) [inline blog/2015/04/SCF_minutes.html]: @karinlag Could you check your last name? From others post is "Lagesen" but here has a extra `b`. Thanks to @arlissc to find this.

Comment (karinlag) [inline blog/2015/04/SCF_minutes.html]: On 04/07/2015 03:57 PM, Raniere Silva wrote:
> In blog/2015/04/SCF_minutes.html
> <https://github.com/swcarpentry/site/pull/920#discussion_r27879491>:
>
>> @@ -0,0 +1,52 @@
>> +---
>> +layout: blog
>> +root: ../../..
>> +author: Karin Lagesebn
>
> @karinlag <https://github.com/karinlag> Could you check your last name?
>  From others post is "Lagesen" but here has a extra |b|. Thanks to
> @arlissc <https://github.com/arlissc> to find this.

Doh. Will fix a bit later tonight.

K

Comment (karinlag) [inline blog/2015/04/SCF_minutes.html]: On 07.04.2015 15:57, Raniere Silva wrote:
> In blog/2015/04/SCF_minutes.html
> <https://github.com/swcarpentry/site/pull/920#discussion_r27879491>:
>
>> @@ -0,0 +1,52 @@
>> +---
>> +layout: blog
>> +root: ../../..
>> +author: Karin Lagesebn
>
> @karinlag <https://github.com/karinlag> Could you check your last name?
>  From others post is "Lagesen" but here has a extra |b|. Thanks to
> @arlissc <https://github.com/arlissc> to find this.

While I remember, is there a way to add a title? I am not posting this 
as me, per se, but rather as the SCF secretary.

Karin

**Comment by karinlag** [comment]
This one is good to go from my side.

**Comment by r-gaia-cs** [comment]
@gvwilson Can you merge this?

**Comment by karinlag** [comment]
On 12/04/15 02:12, Raniere Silva wrote:
> @gvwilson <https://github.com/gvwilson> Can you merge this?

Thanks Raniere, you just beat me to ask Greg to do this :)

Karin

**Comment by gvwilson** [comment]
@r-gaia-cs @karinlag y'all can go ahead and merge pull requests any time :-)

**Comment by karinlag** [comment]
On 12/04/15 16:19, Greg Wilson wrote:
> @r-gaia-cs <https://github.com/r-gaia-cs> @karinlag
> <https://github.com/karinlag> y'all can go ahead and merge pull requests
> any time :-)

:)

I think I am still a bit chicken about this :) But, will unchicken for 
the next one.

karin

**Pull request merged**